IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/wolfcrypt_benchmark.dep

```diff
@@ -0,0 +1,726 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Also the .dep files are generated based on the project file build results and I'm fairly certain doesn't need to be included.